Sources/SwiftSyntaxMacrosTestSupport/Assertions.swift

```diff
@@ -254,16 +254,24 @@ func assertDiagnostic(
 ///   - macros: The macros that should be expanded, provided as a dictionary
 ///     mapping macro names (e.g., `"stringify"`) to implementation types
 ///     (e.g., `StringifyMacro.self`).
+///   - applyFixIts: Applies only the fix-its with these messages.
+///     Ohterwise all Fix-Its from diagnostics will be applied.
+///   - fixedSource: Asserts that the source after applying fix-its matches
 ///   - testModuleName: The name of the test module to use.
 ///   - testFileName: The name of the test file name to use.
+///   - indentationWidth: The indentation width used in the expansion.
+///   - options: Options for asserting fixed source.
 public func assertMacroExpansion(
   _ originalSource: String,
   expandedSource expectedExpandedSource: String,
   diagnostics: [DiagnosticSpec] = [],
   macros: [String: Macro.Type],
+  applyFixIts: [String]? = nil,
+  fixedSource expectedFixedSource: String? = nil,
   testModuleName: String = "TestModule",
   testFileName: String = "test.swift",
   indentationWidth: Trivia = .spaces(4),
+  options: AssertParseOptions = [],
```

Comment:
I would prefer not to expose `AssertParseOptions` for now. It’s always easier to add public API in the future and I don’t think it’s newline normalization is needed to test macros in 99% of all cases. And since we don’t have substructure testing, `substructureCheckTrivia` is irrelevant here as well.